cpython-unix/build-cpython.sh

```diff
@@ -742,7 +774,13 @@ s390x-unknown-linux-gnu)
     PYTHON_ARCH="s390x-linux-gnu"
     ;;
 x86_64-unknown-linux-*)
-    PYTHON_ARCH="x86_64-linux-gnu"
+    # In Python 3.13+, the musl target is identified in cross compiles and the output directory
+    # is named accordingly.
+    if [ "${CC}" = "musl-clang" ] && [ "${PYTHON_MAJMIN_VERSION}" = "3.13" ]; then
+        PYTHON_ARCH="x86_64-linux-musl"
+    else
+        PYTHON_ARCH="x86_64-linux-gnu"
+    fi
```

Comment:
See https://github.com/zanieb/python-build-standalone/pull/6#issuecomment-2330385678 for context.